src/main/java/uk/gov/hmcts/darts/common/repository/TranscriptionRepository.java

```diff
@@ -56,4 +56,11 @@ List<TranscriptionEntity> findByHearingIdTypeStartAndEnd(
         OffsetDateTime startTime,
         OffsetDateTime endTime
     );
+
+    @Query("""
+        SELECT t
+        FROM TranscriptionEntity t
+        WHERE
+        id in :transcriptionIds""")
+    List<TranscriptionEntity> getTranscriptionsForId(List<Integer> transcriptionIds);
```

Comment:
I think it should be possible to just use the `findAllById` offered by the `ListCrudRepository` that we extend from `JpaRepository`

Reply:
renamed to `findByIdIn`